concepts/word-embeddings.md

```diff
@@ -1,7 +1,59 @@
 ---
 grand_parent: Resources
 parent: Concepts
-layout: coming_soon
 title: Word embeddings
 description:
 ---
+
+**Word-embeddings** are a way to represent language words as [vectors](vector.md) of real numbers.
+
+## One-hot encoding
+
+One-hot encoding is one simple method of representing every word from a vocabulary.
+
+One-hot encoding is a unique vector, and its coordinates are mostly 1 and 0.
```

Comment:
One-hot encoding is a unique style of encoding information, where only one position in the vector equals 1, the rest of positions are zeros, therefore the word "one-hot".